### ISSUE TITLE
Kernel: Choose linux 6.6.6 for OS/1337 Version 0.1 "Core" / Floppy Version

### ISSUE THREAD

**Issue by kkarhan** [OS-1337/OS1337#11]
With the release of [Kernel 6.6.6](https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.6.6.tar.xz) . a meme-able distro launch is feasible.

**Comment by kkarhan** [comment]
current ``os1337.img`` [has been updated to Linux 6.6.6-i486](https://github.com/OS-1337/OS1337/raw/main/build/0.CORE/os1337.img) as per [commit](https://github.com/OS-1337/OS1337/commit/04256bfa4cf0f359039a81ae4508bbc8af9c4e72).

**Comment by kkarhan** [comment]
So far it seems to work fine except the non-display of ASCII line drawing characters. 

![Image](https://github.com/OS-1337/OS1337/assets/30401796/c046f411-a3e0-4b71-813b-e1d7411bd41a)

Likely due to [gutted functionality of character device config](https://github.com/OS-1337/OS1337/blob/main/build/0.CORE/build/linux/linux-6.6.6-i486-minimal.config#L747).
See also the [relevant commit](https://github.com/OS-1337/OS1337/commit/59b9383c509cfffeb8982a24604130eea206931b#diff-3f731f167a6e41eb72876e22dfa76758f02242d59c2d4a44b0dd4f119bebe919)

TLDR: at worst a minor issue but nothing worth redoing a lot of stuff for.

**Comment by kkarhan** [comment]
So far, Linux 6.6.6 works fine...
I'd say this can be set to completed.

![Image](https://github.com/OS-1337/OS1337/assets/30401796/c6cb1b8b-edb5-482e-83c0-4a54e3a36c1b)
